egs/chime4/s5_1ch/local/chime4_train_rnnlms.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
```

Comment:
@jtrmal @hainan-xv would either of you have time to go through this file ?
 Add a line note